app/retail/templates/shared/nav.html

```diff
@@ -16,14 +16,16 @@
 
 {% endcomment %}
 {% load i18n static matches cookielaw_tags %}
-<nav class="navbar navbar-expand-md navbar-light bg-white">
-  <a class="d-none d-md-block mr-2 mb-1" href="{% url 'index' %}">
+<nav class="navbar navbar-expand-md navbar-light bg-white p-0 px-md-1">
+  <a class="d-none d-md-block mr-2 mb-1 ml-2" href="{% url 'index' %}">
     {% include 'shared/helmet-purple-animated.html' %}
   </a>
   <i class="black font-bigger-2 d-none d-md-block fa fa-angle-right"></i>
-  <a class="navbar-brand ml-2 py-0" href="/explorer">
-    <!-- <img class="mw-100 logo-static" src="{% static 'v2/images/top-bar/bounties-neg.svg' %}" alt="Bounties" height="35"> -->
-    {% include 'shared/svg-embed/h35px_bounties.svg' %}
+  <a class="navbar-brand sticky-top py-0 mr-0 bg-white my-md-auto" href="/explorer">
```

Comment:
Thanks @chibie, theres quite a few across the whole codebase (I'm seeing 139 across 64 files and then a fair few more in the .js files), I've corrected the ones on the files that I've touched here - but this will have to be an issue in of itself, or we can add it to the tech-debt list and get them corrected as we work on those files.

Resolved in 7cd0d2e3bf96cc58f09dafa0833650d0731f2c7b